mods/cnc/maps/gdi09/rules.yaml

```diff
@@ -5,7 +5,7 @@ World:
 		StartingMusic: march
 		VictoryMusic: gdi_win1
 	MissionData:
-		Briefing: Take out Nod turrets along shore so Gunboats can move in safely on the Nod base.\n\nThe Nod base must be destroyed.\n\nIf gunboats can get in, they should be able to destroy the base with no difficulty.\n\nKeep an eye out for the new weapon Nod is rumored to be working on.
+		Briefing: Take out the Nod turrets along the shore so Gunboats can approach the Nod base.\n\nThe Nod base must be destroyed.\n\nOnce the Gunboats are in range, they should be able to destroy the base with ease.\n\nKeep an eye out for the new weapon rumored to be under development.
```

Comment:
The gunboats may be destroyed before getting into range, so keep the previous wording.